### PR TITLE
(DOCSP-30173): Proactive performance insights

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
## DESCRIPTION

New page explaining proactive performance insights in Compass. I used our [Concept Template](https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCSP-23914/information-types/concept/) for the structure of this page.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-30173/manage-data/performance-insights/

## JIRA

https://jira.mongodb.org/browse/DOCSP-30173

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64badc7787b3b36046d6e21c

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by himanshusinghs — APPROVED** [review]
It is coming out really great. Thanks for this 😄

File: source/manage-data/performance-insights.txt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   :header-rows: 1
+   :widths: 10 10
+
+   * - Scenario
+     - Performance insight
+   
+   * - You run a query or aggregation without an index.
+     - Add an index to support the operation.
+
+   * - You run a pipeline that uses a :pipeline:`$lookup` stage.
```
Comment (kennethdyer) [inline source/manage-data/performance-insights.txt:58]: Question: Would it make sense to qualify "pipeline" as an "aggregation pipeline"?  As a new user unfamiliar with the wider Compass docs/product would I know what a pipeline is without the qualifier?

**Review by kennethdyer — APPROVED** [review]
Thanks for this @jeff-allen-mongo.  Have a non-blocking question but otherwise it LGTM.

File: source/manage-data/performance-insights.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/manage-data/performance-insights.txt]: Agreed. Updated.

**Pull request merged**